### PR TITLE
Add 'show-commit-details' feature

### LINKED ISSUE #3277: Show latest commit message on project page
The most recent Github update hides the latest commit message on a project page. Instead, it only shows the author, commit date, and SHA:

<image src="https://imgur.com/wDLHkZn.png">

This seems like a misfeature to me - there's always enough space to display at least part of the commit message. Seeing a commit message by default is much more useful than a SHA - I have no reason to want to copy a SHA until I know what the commit does.

I think it would be useful to hide the SHA and use the extra space to display as much of the commit message as will fit.

### PR DESCRIPTION
**Pull request by Aaron1011**
1. LINKED ISSUES: Closes #3277

2. TEST URLS: https://github.com/sindresorhus/refined-github

3. SCREENSHOT:  
refined_comment

### CODE REVIEW

File: source/features/show-commit-details.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function init(): void {
+	// Ideally, we would run this immediately on page load, but there
+	// seems to be a race condition. Delaying by 100ms makes it work
+	// consistently.
+	setTimeout(() => {
```
Comment (kidonng) [inline source/features/show-commit-details.tsx:10]: I think you should use `element-ready` instead.

Comment (Aaron1011) [reply source/features/show-commit-details.tsx:10]: The button exists in the HTML sent by the server. I think there are other elements that need to be created before the click will work, but I haven't been able to determine what they are.

**Comment by busches** [comment]
What does it look like on a commit with more than one line in the message?

Did you consider a css solution?

**Comment by kidonng** [comment]
> Did you consider a css solution?

```css
.Details:not(.Details--on) .Details-content--hidden {
  display: block !important;
  height: 1.5em;
  overflow: hidden;
}
```

**Comment by fregante** [comment]
Wait for #3277 to be accepted first, because it might not

**Comment by fregante** [comment]
Can you implement this via CSS as described in https://github.com/sindresorhus/refined-github/issues/3277 ?

**Pull request closed without merging**